### PR TITLE
Add and Remove Functions

### PR DESCRIPTION
**Pull request by stino-x**
added necessary add and remove functionality to to do list

### CODE REVIEW

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @stino-x, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has failing linters, kindly fix this.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/Modules/Delete.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import AddToScreen from './script.js';
+import AddToScreen from '../index.js';
 import { saveTaskstoLocalStorage } from './AddtoLocalStorage.js';
```
Comment (Munsa1) [inline src/Modules/Delete.js:2]: - [ ] The main requirement for this milestone is to be able to delete items individually, The three dots to the right of the task should change to a `delete` icon and upon being clicked and should delete the Item. kindly implement this. Also the user should be able to edit task descriptions, upon adding tasks as well, kindly fix this
image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @stino-x, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Passing Linters :heavy_check_mark: 
Correct GitHub Flow :heavy_check_mark: 
Descriptive ReadMe File :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._
- [ ] Kindly fix the broken link to your LinkedIn profile in the ReadMe file
image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Modules/Delete.js (hunk 1 above)
Comment (didierganthier) [reply src/Modules/Delete.js:2]: Hey @stino-x 👋🏽 as requested by the previous reviewer [here](https://github.com/stino-x/check-list/pull/3/files#r1193061553) the main requirement for this milestone is to be able to delete items individually, The three dots to the right of the task should change to a delete icon and upon being clicked should delete the Item. 

Upon further testing, I noticed that it's doing that only when I double-tap the three dots. Please adjust that to meet the project requirements. 💯

Comment (stino-x) [reply src/Modules/Delete.js:2]: @Munsa1 the functionality is working double click on the three dots

**Review by didierganthier — CHANGES_REQUESTED** [review]
N/A

File: src/Modules/Delete.js (hunk 1 above)
Comment (didierganthier) [reply src/Modules/Delete.js:2]: [OPTIONAL] Hey @stino-x  👋🏽 I'm marking that as optional as this is not a requirement that it must be done exactly like that but in order to improve the user experience consider the comment of the previous reviewer. 👍🏽

**Review by didierganthier — APPROVED** [review]
Hi @stino-x ,

Congratulations! 🎉 Your project is complete and ready to be merged! 🚀



## Highlights
- Fixed broken link ✔️ 
- Add/remove functionality ✔️ 

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.

Cheers, and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread, and don't forget to tag me so I'll receive the notification.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Once again, congratulations on completing this project! 🎉

**Pull request merged**